wad.md

```diff
@@ -82,14 +82,15 @@ WAD versions 2 and 3 include all fields.
 |  12 |    4 | u32    | compressed size                        |
 |  16 |    4 | u32    | uncompressed size                      |
 |  20 |    1 | u8     | compression type                       |
-|  21 |    1 | bool   | *unknown*                              |
-|  22 |    2 | u16    | *unknown*                              |
+|  21 |    1 | bool   | [is duplicated ?](#entry-duplication)  |
+|  22 |    2 | u16    | padding                                |
 |  24 |    8 | u64    | first 8 bytes of sha256 hash of data   |
 
 The following *compression types* are known:
 
  - `0` -- uncompressed data
  - `1` -- zlib
+ - `2` -- [file redirection](#file-redirection)
```

Comment:
Since the *file redirection* type is not a compression, *compression type* should be renamed to *type* in the doc.

Reply:
done